skimage/util/compare.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+from ..util import img_as_float64
+from itertools import product
+
+
+def compare_images(image1, image2, method='diff'):
+    """
+    Return an image showing the differences between two images.
+
+    Parameters
+    ----------
+    image1, image2 : 2-D array
+        Images to process, must be of the same shape.
+    method : string, optional
+        Method used for the comparison.
+        Valid values are {'diff', 'blend', 'checkerboard'}
```

Comment:
I think a short explanation of what each mode does would be useful. 'blend' and 'checkerboard' may not be immediately obvious to everyone. I think that would combine nicely with an examples section.